pkg/registry/apis/dashboard/register.go

```diff
@@ -304,12 +345,12 @@ func (b *DashboardsAPIBuilder) validateCreate(ctx context.Context, a admission.A
 	// Validate folder existence if specified
 	if !a.IsDryRun() && accessor.GetFolder() != "" {
 		if err := b.validateFolderExists(ctx, accessor.GetFolder(), id.GetOrgID()); err != nil {
-			return apierrors.NewNotFound(folders.FolderResourceInfo.GroupResource(), accessor.GetFolder())
+			return err
 		}
 	}
 
 	// Validate quota
-	if !a.IsDryRun() {
+	if !b.isStandalone && !a.IsDryRun() {
```

Comment:
Could you add a ticket for this? We will need to implement quotas in either mode

Reply:
sure!